dgraphtest/dgraph.go

```diff
@@ -290,6 +290,11 @@ func publicPort(dcli *docker.Client, dc dnode, privatePort string) (string, erro
 	return "", fmt.Errorf("no mapping found for private port [%v] for container [%v]", privatePort, dc.cname())
 }
 
+// Wrapper function to export publicport() in other files
```

Comment:
we shouldn't add this function

Reply:
Resolved

Reply:
how is it resolved? The function is still exported